config/environments/production.rb

```diff
@@ -19,7 +19,7 @@
   config.public_file_server.enabled = false
 
   # Compress JavaScripts and CSS.
-  config.assets.js_compressor = :uglifier
+  config.assets.js_compressor = Uglifier.new(harmony: true)
```

Comment:
@nofaralfasi can you also add some es6 features across multiple pages to show that everything is still working?
e.g. `let, const, arrow function`